rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
+
```

Comment:
- [ ] In the Rental class, the @book and @person instance variables should have an attr_accessor present